.github/workflows/gcr-cleaner.yaml

```diff
@@ -0,0 +1,32 @@
+# .github/workflows/gcr-cleaner.yaml
+name: 'gcr-cleaner'
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - 'v*'
+jobs:
+  gcr-cleaner:
+    runs-on: 'ubuntu-latest'
+    steps:
+      # configure based on registry
+      - uses: 'docker/login-action@v2'
```

Comment:
We're using `docker/login-action@v1` in our container builds:

https://github.com/trisacrypto/directory/blob/main/.github/workflows/containers.yaml#L55

Do you think we should upgrade our other workflows to use `v2`?

Reply:
perhaps we can update all the workflows together at another time. I'll create a story.